openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -175,95 +175,102 @@ The following subsections illustrate some of the authorization flows supported b
+Note: Steps (3) and (4) happen in the front channel, by redirecting the End-User via the User Agent. Those steps are defined in (#authorization_endpoint). The Authorization Server and the User Agent may exchange any further communication, the actual user authentication is out of scope for this specification.
 
-(5) The Wallet sends a Credential Request to the Credential Issuer's Credential Endpoint with the Access Token and (optionally) the proof of possession of the public key to which the issued VC shall be bound to. Upon successfully validating Access Token and proof, the Credential Issuer returns a VC in the Credential Response if it can issue a Credential right away. This step is defined in (#credential-endpoint).
+(4) The Authorization Endpoint returns an Authorization Response with the Authorization Code upon successfully processing the Authorization Request.
 
-If the Credential Issuer requires more time to issue a Credential, the Credential Issuer MAY return a `transaction_id` to the Wallet with information about when the Wallet can start sending a Deferred Credential Request to obtain an issued Credential, as defined in (#deferred-credential-issuance).
+(5) The Wallet sends a Token Request to the Token Endpoint with the Authorization Code obtained in step (4). The Token Endpoint returns an Access Token in the Token Response upon successfully validating Authorization Code. This step happens in the back-channel communication (direct communication between two systems using HTTP requests and responses without using redirects through an intermediary such as a browser). This step is defined in (#token_endpoint).
 
-If the Credential Issuer wants to issue multiple Credentials in one response, the Credential Issuer MAY support the Batch Credential Endpoint and the Wallet MAY send a Batch Credential Request to the Batch Credential Endpoint as defined in (#batch-credential-endpoint).
+(6) The Wallet sends a Credential Request to the Credential Issuer's Credential Endpoint with the Access Token and (optionally) the proof of possession of the public key to which the issued (Verifiable) Credential shall be bound to. Upon successfully validating Access Token and proof, the Credential Issuer returns a Credential in the Credential Response. This step is defined in (#credential-endpoint).
 
-With grant type `authorization_code`, it is RECOMMENDED to use PKCE as defined in [@!RFC7636] to prevent authorization code interception attacks and Pushed Authorization Requests [@RFC9126] to ensure integrity and authenticity of the authorization request.
+If the Credential Issuer requires more time to issue a Credential, the Credential Issuer may return a Transaction ID and a time interval to the Wallet in the Credential Response. After the specified time has passed, the Wallet may send a Deferred Credential Request with the Transaction ID to obtain a Credential, as defined in (#deferred-credential-issuance).
```

Comment:
```suggestion
If the Credential Issuer cannot provide the Credential directly in the response, in the cases where it requires more time, the Credential Issuer MAY return in the Credential Response a `Transaction ID` and a time interval, beyond which the Wallet MAY send a Deferred Credential Request using the `Transaction ID` to obtain the Credential, as defined in (#deferred-credential-issuance).
```

Reply:
@Sakurann @tlodderstedt Is section 3 more of an informative overview or shall it contain normative statements with capialized verbs, i.e. MAY or may?

Reply:
i think it is ok for section 3 to contain normative MAYs

Reply:
I will keep the text as-is then